packages/cli/src/util/commands-config.ts

```diff
@@ -16,8 +16,6 @@ export const applyCommandsConfig = (command: Command, config: CommandConfig) =>
     config.arguments.map((argument: any) => {
       if (argument.flags === '<packageRefs...>') {
         command.argument(argument.flags, argument.description, parsePackagesArguments, argument.defaultValue);
-      } else if (command.name() === 'interact' && argument.flags === '<packageRef>') {
-        command.argument(argument.flags, argument.description, parsePackageArguments, argument.defaultValue);
```

Comment:
Small refactor, as the `parsePackageArguments` is intended for parsing settings, but in `interact` we don't need to parse settings.